Day01/app.py

```diff
@@ -11,11 +11,8 @@
 # Apro il file in lettura
 with open(path, "r") as file:
     riga = file.readline()
-    """Le righe di sotto iterano sul file cercando di trovare i numeri ma purtroppo trovano dei numeri 
-    che non sono presenti nel file, quindi non riesco a trovare i numeri giusti"""
     for riga in file:
-        numeri = [numero for numero in lista_numeri if str(numero) in riga]
-        print(numeri)
+        print("Stiamo valutando la riga %s" % (riga))
```

Comment:
Il loop per leggere le righe è corretto, infatti se proviamo a stampare ogni riga viene fuori correttamente.
Il prossimo passo sarebbe leggere ogni singolo carattere, verificare che sia una cifra e non una lettera e poi salvarli in una lista